app/javascript/tagging/components/InnerComponents/ValueModifier.js

```diff
@@ -11,19 +11,22 @@ const ValueModifier = ({
   multiValue,
   valueLabel,
   isDisabled,
-}) => (
-  <FormGroup legendText={valueLabel}>
-    <Column className="tag-modifier-form-row value-modifier">
-      <ValueSelector
-        values={values}
-        onTagValueChange={onTagValueChange}
-        selectedOption={selectedTagValues}
-        multiValue={multiValue}
-        isDisabled={isDisabled}
-      />
-    </Column>
-  </FormGroup>
-);
+}) => {
+  console.log('selectedTagValues=', selectedTagValues);
```

Comment:
@jeffibm Other than this console.log the rest of the pr is good. I can merge once this is removed.